state/runtime/evm/instructions_test.go

```diff
@@ -10,81 +10,245 @@ import (
 	"github.com/0xPolygon/polygon-edge/state/runtime"
 	"github.com/0xPolygon/polygon-edge/types"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
 	"github.com/stretchr/testify/require"
 )
 
 var (
-	two = big.NewInt(2)
+	two   = big.NewInt(2)
+	three = big.NewInt(3)
+	four  = big.NewInt(4)
+	five  = big.NewInt(5)
 
 	allEnabledForks = chain.AllForksEnabled.At(0)
 )
 
-type twoOperandsArithmetic []struct {
-	a              *big.Int
-	b              *big.Int
-	expectedResult *big.Int
+type OperandsLogical []struct {
```

Comment:
Just thinking outloud:
is it better to define `OperandsLogical` and `OperandsArithmetic` to be simple type, rather than a slice?

```suggestion
type OperandsLogical struct {
```

```suggestion
type OperandsArithmetic struct {
```